CMakeLists.txt

```diff
@@ -2064,6 +2064,46 @@ target_link_libraries(mixxx-lib PUBLIC
   Qt5::Widgets
   Qt5::Xml)
 target_compile_definitions(mixxx-lib PUBLIC QT_TABLET_SUPPORT QT_USE_QSTRINGBUILDER)
+get_target_property(QT5_TYPE Qt5::Core TYPE)
+if(QT5_TYPE STREQUAL "STATIC_LIBRARY")
+
+  # NOTE(rryan): If you are adding a plugin here, you must also
+  # update src/mixxxapplication.cpp to define a Q_IMPORT_PLUGIN
+  # for it. Not all imageformats plugins are built as .libs when
+  # building Qt statically on Windows. Check the build environment
+  # to see exactly what's available as a standalone .lib vs linked
+  # into Qt .libs by default.
+
+  set_property(TARGET mixxx-lib APPEND PROPERTY INTERFACE_LINK_LIBRARIES
```

Comment:
```suggestion
  target_link_libraries(mixxx-lib PRIVATE
```

Reply:
ping

Reply:
Why do you think your proposal is better?
What is the difference?

Reply:
They might be the same functionally, but this is an unusual way of doing it.

Reply:
Done